cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +481,55 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+def test_migration_long_name(request, appliance, v2v_providers, host_creds, conversion_tags):
+    """Test to check VM name with 64 character should work"""
+    source_datastores_list = v2v_providers[0].data.get('datastores', [])
+    source_datastore = [d.name for d in source_datastores_list if d.type == 'nfs'][0]
+    # Following code will create vm name with 64 characters
+    vm_name = random_vm_name(context="v2v") + fauxfactory.gen_alpha(51)
+    collection = v2v_providers[0].appliance.provider_based_collection(v2v_providers[0])
+    vm_obj = collection.instantiate(name=vm_name,
+                                    provider=v2v_providers[0],
+                                    template_name=rhel7_minimal(v2v_providers[0])['name'])
+    vm_obj.create_on_provider(timeout=2400,
+                              find_in_cfme=True,
+                              allow_skip="default",
+                              datastore=source_datastore)
+    request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
+    form_data = _form_data(v2v_providers[0], v2v_providers[1])
+
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    mapping = infrastructure_mapping_collection.create(form_data)
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    migration_plan = migration_plan_collection.create(
+        name="plan_long_name{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_long_name{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping.name,
+        vm_list=[vm_obj],
+        start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    # wait until plan is in progress
+    wait_for(func=view.plan_in_progress, func_args=[migration_plan.name],
+        message="migration plan is in progress, be patient please",
+        delay=5, num_sec=1800)
+
+    view.migr_dropdown.item_select("Completed Plans")
+    view.wait_displayed()
+    logger.info("For plan %s, migration status after completion: %s, total time elapsed: %s",
```

Comment:
Optional:
old `printf` format used; you can go with new string format style. I know officially python not oppose to old formatting but still, we to avoid error new one is best.

Reply:
Beware of rest `test_v2v_migration.py` contains existing same `%s` formats